Add endpoints docs to wiki (#883)
iota-wiki/iota-wiki: iota/develop/endpoints/devnet.md

---
description: The IOTA foundation provides load-balanced public devnet endpoints, where MQTT and the HTTP REST API are enabled.
image: /img/logo/Chrysalis_logo_dark.png
keywords:
- devnet
- load-balanced
- HTTP REST API
- MQTT
- reference
- Endpoints
---
# Devnet

Since the Chrysalis update, the `testnet` is now called `devnet`.  We recommend using the `devnet` to develop and test your application.  

## Public Infrastructure

> IOTA currently provides a load-balanced public devNet endpoint:

Author: ```suggestion
IOTA currently provides a load-balanced public devnet endpoint:
```

- [https://api.lb-0.h.chrysalis-devnet.iota.cafe/](https://api.lb-0.h.chrysalis-devnet.iota.cafe/).
- [https://api.lb-1.h.chrysalis-devnet.iota.cafe/](https://api.lb-1.h.chrysalis-devnet.iota.cafe/).

:::note

We recommend using the load balancer for most scenarios.

:::

We also provide single node endpoints that expose native [MQTT](https://mqtt.org/):

- [https://api.thin-hornet-0.h.chrysalis-devnet.iota.cafe](https://api.thin-hornet-0.h.chrysalis-devnet.iota.cafe)
- [https://api.thin-hornet-1.h.chrysalis-devnet.iota.cafe](https://api.thin-hornet-1.h.chrysalis-devnet.iota.cafe)
- mqtt.lb-0.h.chrysalis-devnet.iota.cafe:1883
- mqtt.lb-1.h.chrysalis-devnet.iota.cafe:1883

These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this [specification](https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/rest-api.yaml)) over TLS.

## Developer Tools

- [Explorer](https://explorer.iota.org/devnet).
- [Online Faucet](https://faucet.chrysalis-devnet.iota.cafe).

> - [cli-wallet](https://github.com/iotaledger/cli-wallet).
> - [chrysalis-faucet Code (nodejs + svelte)](https://github.com/iotaledger/chrysalis-faucet).

Author: As this are endpoints docs and we already mention the cli-wallet under tools in the `Develop` section I think we can remove this.
```suggestion
```

Reviewer: Agreed

---
description: The IOTA foundation provides load-balanced public devnet endpoints, where MQTT and the HTTP REST API are enabled.
image: /img/logo/Chrysalis_logo_dark.png
keywords:
  - devnet
  - load-balanced
  - HTTP REST API
  - MQTT
  - reference
  - Endpoints
---

# Devnet

Since the Chrysalis update, the `testnet` is now called `devnet`. We recommend using the `devnet` to develop and test your application.

## Public Infrastructure

> IOTA currently provides a load-balanced public devnet endpoint:

Reviewer 2: ```suggestion
The IOTA Foundation currently provides the following public load-balanced devnet endpoints:
```

- [https://api.lb-0.h.chrysalis-devnet.iota.cafe/](https://api.lb-0.h.chrysalis-devnet.iota.cafe/).
- [https://api.lb-1.h.chrysalis-devnet.iota.cafe/](https://api.lb-1.h.chrysalis-devnet.iota.cafe/).

:::note

We recommend using the load balancer for most scenarios.

:::

We also provide single node endpoints that expose native [MQTT](https://mqtt.org/):

- [https://api.thin-hornet-0.h.chrysalis-devnet.iota.cafe](https://api.thin-hornet-0.h.chrysalis-devnet.iota.cafe)
- [https://api.thin-hornet-1.h.chrysalis-devnet.iota.cafe](https://api.thin-hornet-1.h.chrysalis-devnet.iota.cafe)
- mqtt.lb-0.h.chrysalis-devnet.iota.cafe:1883
- mqtt.lb-1.h.chrysalis-devnet.iota.cafe:1883

> These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this [specification](https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/rest-api.yaml)) over TLS.

Reviewer: Should we point to the Wiki spec now?

```suggestion
These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this [specification](nodes/api_reference.md)) over TLS.
```

Reviewer: Same here, maybe this is better to save one click.

```suggestion
These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this [specification](nodes/rest-api/iota-rest-api.info.mdx)) over TLS.
```
